src/dialogs/more-info/ha-more-info-dialog.ts

```diff
@@ -57,7 +74,9 @@ export class MoreInfoDialog extends LitElement {
     this.large = false;
     this._stateHistory = undefined;
     if (this._computeShowHistoryComponent(this._entityId)) {
+      this._fetchUserDone = this._fetchUserNames();
       this._getStateHistory();
+      this._getLogBookData();
```

Comment:
We should fetch this data when the history tab is activated. Extract all history and logbook code and bring it to a new component `ha-dialog-tab-history`. We should then lazy load that component after the dialog is opened.